digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -375,6 +375,10 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
     }
   }
 
+  private def isRoadPartTransfer(projectLinks: Seq[ProjectLink], roadNumber: Long , newRoadPart: Long): Boolean = {
+    projectLinks.filter(_.roadPartNumber == newRoadPart).length == 0 && projectLinks.filter(_.roadNumber == roadNumber).length > 0
```

Comment:
.exists() is superior to .filter...length

Reply:
What is the check that this method actually does?
There is a link in this sequence where this newRoadPart is not mentioned but this roadNumber is?